lib/auth/auth_with_roles_test.go

```diff
@@ -4245,13 +4258,13 @@ func TestListUnifiedResources_WithSearch(t *testing.T) {
 		SortBy:         types.SortBy{IsDesc: true, Field: types.ResourceMetadataName},
 	})
 	require.NoError(t, err)
-	require.Len(t, resp.Resources, 2)
+	require.Len(t, resp.Resources, 1)
 	require.Empty(t, resp.NextKey)
 
 	// Check that our returned resource has the correct name
 	for _, resource := range resp.Resources {
-		r := resource.GetNode()
-		require.True(t, strings.Contains(r.GetHostname(), "tifa"))
+		r := resource.GetAppServerOrSAMLIdPServiceProvider()
+		require.True(t, strings.Contains(r.GetName(), "tifa"))
```

Comment:
I see "tifaSAML", but where does the "tifa" name come from?

Reply:
I'm using "tifa" here to test the search functionality. Basically "can i type some string and get resources that include that substring in their name"

Reply:
I somehow missed the "contains". Thanks.